### PR TITLE
Release 0.14.0 with fixes to previous PR

### PR DESCRIPTION
**Pull request by pksunkara**

### CODE REVIEW

File: lib/primitives/base-element.js [hunk 1]
```diff
@@ -101,7 +98,7 @@ module.exports = function(registry) {
       var meta = {};
 
       this.meta.keys().forEach(function (key) {
-        this.convertKeyToRefract(meta, key, this.meta.get(key));
+        meta[key] = this.convertKeyToRefract(key, this.meta.get(key));
```
Comment (smizell) [inline lib/primitives/base-element.js:101]: This is a good change. Nice work.

File: lib/primitives/base-element.js [hunk 2]
```diff
@@ -114,13 +111,13 @@ module.exports = function(registry) {
...
       this.attributes.keys().forEach(function (key) {
-        this.convertKeyToRefract(attributes, key, this.attributes.get(key));
+        attributes[key] = this.convertKeyToRefract(key, this.attributes.get(key));
       }, this);
 
       return attributes;
     },
 
-    fromRefract: function(doc) {
+    fromRefract: function(doc, options) {
```
Comment (smizell) [inline lib/primitives/base-element.js:120]: Naming question: is there a better name for this? This is not really for options for the functionality, but rather an object to merge into the newly-created element.

Comment (pksunkara) [inline lib/primitives/base-element.js:120]: It is named as `options` in `toRefract` too.

**Comment by smizell** [comment]
@pksunkara I just have one comment. The naming of the variable did not really convey what was going on to me, so I think it would be helpful to name it something different.

Other than that, this looks good to me. If we can address that, I'll merge.

File: lib/primitives/base-element.js (hunk 2 above)
Comment (smizell) [inline lib/primitives/base-element.js:120]: Ok, let's leave then.

**Pull request merged**